css/index.css

```diff
@@ -1,3 +1,342 @@
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    /* font-family: 'Titillium Web', sans-serif; */
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+  margin: 20px auto;
+  width: 768px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header .logo {
+}
+
+header nav {
+  display: flex;
+  justify-content: flex-end;
+  align-content: center;
+}
+
+header a {
+  text-decoration: none;
+  background: lightblue;
+  color: gray;
+  line-height: 3;
+  margin-left: 20px;
+  margin-right: 20px;
+  display: inline-block;
+}
+
+.mainheading {
+  height: 435px;
+  margin-top: 100px;
+  display: flex;
+  position: relative;
+  border-bottom: 3px solid black;
+}
+
+.mainheading .box3a {
+  width: 436px;
+  height: 435px;
+  text-transform: uppercase;
+  font-size: 40px;
+  text-align: center;
+  display: inline-block;
+  left: 0;
+  padding-top: 70px;
+}
+
+.mainheading .box3a button {
+  font-size:50%;
+  background: white;
+  border-color: black;
+  border-width: 1px;
+  border-style: ridge; 
+}
+
+.mainheading .box3b {
+ width: 435px;
+ right: 0;
+}
+
+.top-section {
+  height: 163px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+}
+
+.top-section .box4 {
+  width: 440px;
+}
+
+.top-section .box4 h2 {
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+.top-section .box5 {
+  width: 435px;
+}
+
+.top-section .box5 h2{
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+.middle-img{
+  width: 876px;
+  height: 182px;
+  object-fit: cover;
+}
+
+.box-6 {
+  padding: 10px;
+  margin: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.bottom-section {
+  border-bottom: 3px solid black;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  height: 180px;
+}
+
+.bottom-section .bottom-child {
+  width: 279px;
+  display: inline-block;
+  position: relative;
+}
+
+.bottom-section .bottom-child h2{
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+.contact {
+  line-height: 2;
+}
+
+.box-10 {
+  display: inline-block;
+  position: relative;
+}
+.container3 h2 {
+  font-weight: bold;
+}
+
+.box-11 {
+  padding: 10px 0px 0px 300px;
+}
+
+
+@media (max-width: 800px) {
+  
+  .container {
+    font-size: small; 
+  }
```

Comment:
Shrinking the font size isn't a good practice for adapting to break points. Your font was pretty small to begin. You want the lay out to change, not the font size to shrink.